doc/sphinx/docs/python/tutorials/using_archipelago.rst

```diff
@@ -109,7 +109,7 @@ After inspection, let us now run the evolution.
     array([  4.07802374, -23.82020921,  -4.07802374]),
     array([  1.71396489, -25.90794514,  -1.71396489]),
     ...]
-    >>> archi.evolve()
+    >>> archi.evolve() #doctest: +SKIP
```

Comment:
Why do this (and in a bunch of places below) need to be skipped as no output is being produced?